docs/_articles/en/apex/writing.md

```diff
@@ -25,7 +25,57 @@ Out of the box snippets for Salesforce development are available in these reposi
 - [Apex Code Snippets](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex/snippets/apex.json)
 - [Code Snippets for LWC development](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-lwc/snippets/lwc.json) 
 
-If you want to define your own code snippets, read [Snippets in Visual Studio Code](https://code.visualstudio.com/docs/editor/userdefinedsnippets).
+
+### Example Custom Snippet
+The real power of snippets lies in being able to customize snippets for your own use. Follow these steps to create a custom Apex snippet that lets you quickly write a simple SOQL query:
+
+1. Run the `SFDX: Configure User Snippets` command from the Command Palette. 
```

Comment:
@sbudhirajadoc - I'd have to take a look but don't we use bold for menu command to mirror uicontrol? Or did you change the style?

Reply:
Thanks for catching that. Those commands need to be in bold. I'll make the udpate. Thanks!